### ISSUE TITLE
php 8

### ISSUE THREAD

**Issue by rolandverner** [jpmurray/laravel-countdown#10]
[InvalidArgumentException]                                                                                                         
  Package jpmurray/laravel-countdown has a PHP requirement incompatible with your PHP version, PHP extensions and Composer version:  
    - jpmurray/laravel-countdown requires php ^7.0 which does not match your installed version 8.0.8.                                
    - jpmurray/laravel-countdown requires nesbot/carbon ^2.0 but it is not present.                                                  
    - jpmurray/laravel-countdown requires illuminate/support 6.* || 7.* || 8.* but it is not present.

**Comment by jpmurray** [comment]
Yeah, it's not been tested for PHP 8. I'll see if I can see that do that this month, but meanwhile, if you are able to test and PR, it'd be welcomed !
